app/processing/components/get-schedule-dates.js

```diff
@@ -28,6 +30,7 @@ const mapSchedule = (schedule) => {
   return schedule.map((segment, i) => ({
     order: i + 1,
     dueDate: segment.dueDate?.format('DD/MM/YYYY'),
+    paymentType: segment.paymentType ? segment.paymentType : 'Quarterly payment',
```

Comment:
Could simplify to segment.paymentType ?? 'Quarterly payment'.

Also, conscious over this magic string.

I would suggest moving it into a new constant file and importing it -- it took me a long time to work out this is the default payment period if it's not `Immediate payment` (which I'm also conscious of being a magic string -- the fact we had this change to make means we had to change it in multiple files (here and its tests), however if we had it stored in the constant file and both files imported it, would only be the 1 place))